src/components/views/settings/tabs/room/SecurityRoomSettingsTab.js

```diff
@@ -294,34 +298,38 @@ export default class SecurityRoomSettingsTab extends React.Component {
                     {_t('Changes to who can read history will only apply to future messages in this room. ' +
                         'The visibility of existing history will be unchanged.')}
                 </div>
-                <label>
-                    <input type="radio" name="historyVis" value="world_readable"
-                           disabled={!canChangeHistory}
-                           checked={history === "world_readable"}
-                           onChange={this._onHistoryRadioToggle} />
+                <StyledRadioButton
+                    name="historyVis" value="world_readable"
+                    disabled={!canChangeHistory}
+                    checked={history === "world_readable"}
+                    onChange={this._onHistoryRadioToggle}
+                >
                     {_t("Anyone")}
-                </label>
-                <label>
-                    <input type="radio" name="historyVis" value="shared"
-                           disabled={!canChangeHistory}
-                           checked={history === "shared"}
-                           onChange={this._onHistoryRadioToggle} />
+                </StyledRadioButton>
+                <StyledRadioButton
+                    name="historyVis" value="shared"
+                    disabled={!canChangeHistory}
+                    checked={history === "shared"}
+                    onChange={this._onHistoryRadioToggle}
+                >
                     {_t('Members only (since the point in time of selecting this option)')}
-                </label>
-                <label>
-                    <input type="radio" name="historyVis" value="invited"
-                           disabled={!canChangeHistory}
-                           checked={history === "invited"}
-                           onChange={this._onHistoryRadioToggle} />
+                </StyledRadioButton>
+                <StyledRadioButton
+                    name="historyVis" value="invited"
+                    disabled={!canChangeHistory}
+                    checked={history === "invited"}
+                    onChange={this._onHistoryRadioToggle}
+                >
                     {_t('Members only (since they were invited)')}
-                </label>
-                <label >
-                    <input type="radio" name="historyVis" value="joined"
-                           disabled={!canChangeHistory}
-                           checked={history === "joined"}
-                           onChange={this._onHistoryRadioToggle} />
+                </StyledRadioButton>
+                <StyledRadioButton
+                    name="historyVis" value="joined"
+                    disabled={!canChangeHistory}
+                    checked={history === "joined"}
+                    onChange={this._onHistoryRadioToggle}
+                >
                     {_t('Members only (since they joined)')}
-                </label>
+                </StyledRadioButton>
```

Comment:
same as above

Reply:
I had to extend the radio buttons a little to make it work